docs/source/algorithms/EnggFitPeaks-v1.rst

```diff
@@ -33,11 +33,38 @@ then the algorithm fails.
 
 After the conversion of units, the algorithm tries to fit (in
 time-of-flight) a peak in the neighborhood of every expected peak
-using a peak shape or function. The conversion is done as in the
-Mantid algorithm :ref:`AlignDetectors <algm-AlignDetectors>`
-(following GSAS equations) if the workspace is focused (single
+using a peak shape or function. If the workspace is focused (single
 spectrum) and has a log entry named "difc", where the GSAS DIFC
-parameter is expected. Otherwise the conversion of units is done as in
+parameter is expected, the conversion is done following GSAS equations.
+The equation in `GSAS
+<https://subversion.xor.aps.anl.gov/trac/pyGSAS>`_ converts from
+d-spacing (:math:`d`) to time-of-flight (:math:`TOF`) by the equation:
+
+.. math:: TOF = DIFC * d + DIFA * d^2 + TZERO
+
+The manual describes these terms in more detail. Roughly,
+:math:`TZERO` is related to the difference between the measured and
+actual time-of-flight base on emission time from the moderator, :math:`DIFA` is an empirical term (ideally zero), and :math:`DIFC` is
+
+.. math:: DIFC = \frac{2m_N}{h} L_{tot} sin \theta
```

Comment:
I think this info should actually live in unit factory docs
https://docs.mantidproject.org/nightly/concepts/UnitFactory.html#unit-factory
or in the `ConvertUnits` docs.
This algorithm is no longer used (it pre-dates my work on engineering GUI) - also the name suggest something engineering specific whereas this relates to TOF powder diffraction.

Reply:
Does ConvertUnits actually use `DIFC`? I can't see an obvious place to put it in the documentation.